### ISSUE TITLE
Prediction with MINIROCKET

### ISSUE THREAD

**Issue by MKiremitci** [timeseriesAI/tsai#111]
Hi, first of all thanks to you for this great work. I want to predict time-series samples with MINIROCKET that implemented with Pytorch. Can I do that? How? I am not good at coding, can you help me?

**Comment by oguiza** [comment]
Hi @MKiremitci,
Thanks for your comment. 
I'd recommend you take a look at the {MiniRocket tutorial notebook](https://github.com/timeseriesAI/tsai/blob/main/tutorial_nbs/10_Time_Series_Classification_and_Regression_with_MiniRocket.ipynb) we built to demonstrate how to use it in time series classification and time series prediction.

**Comment by MKiremitci** [comment]
@oguiza Thans for your reply. Actually, I look at it, but I couldn't understand which part of the code predict. 

I think this is the training part.
```
# As above, use tsai to bring X_feat into fastai, and train.
tfms = [None, TSClassification()]
batch_tfms = TSStandardize(by_sample=True)
dls = get_ts_dls(X_feat, y, splits=splits, tfms=tfms, batch_tfms=batch_tfms)
model = build_ts_model(MiniRocketHead, dls=dls)
learn = Learner(dls, model, metrics=accuracy, cbs=ShowGraph())
timer.start()
learn.fit_one_cycle(10, 3e-4)
timer.stop()
```

And, this is the evaluation part.
`
learn.validate()
`

However I couldn't understand the prediction part. Really sorry for my inexperience and for bad English :(

**Comment by dnth** [comment]
Hi @MKiremitci TSAI is built on top of fast.ai library. I understand your struggle as I have went through it myself. However, [learning fastai](https://www.youtube.com/watch?v=_QUEXsHfsA0&list=PLfYUBJiXbdtRL3FMB3GoWHRI8ieU6FhfM&index=1&ab_channel=JeremyHoward) would significanly help you understand the codes here.

As to you question, there are few methods you can use to get the predictions ie `learn.predict(item)` and `learn.get_preds(ds_idx)`
All other methods are listed [here](https://docs.fast.ai/learner.html#Inference-functions).

For example, if you run `learn.get_preds(ds_idx=0)` you will get two variables that contain a list of the predicted value and the training set label.

If you would like to predict on data that is not in your train/validation/test set you can use the `learn.predict(item)` where `item` is the data that you would like to run the prediction on. The documentation on how to use it is on the fastai documentation page as linked above. 

I am also still learning myself please correct me if anyone sees a mistake in my understanding. Hope it helps :)

**Comment by MKiremitci** [comment]
Hi @dnth, Thanks you so much. I actually solved how can I predict on a sample. However, your answer can help the other like me who don't understand how to predict. I am closing this issues. Thank all of you for your replies again. :) Have a nice day.
